public/client-v2/src/components/CAPI/CAPISearchQuery.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import * as React from 'react';
+import capiQuery from '../../services/capiQuery';
+import Async, { type AsyncChild } from '../util/Async';
+
+type ImageAsset = {
```

Comment:
I'd image a lot of the types defined here, such as images and articles, are going to be used across the application (clipboard, rendering collections) so maybe they should live in a separate file?

Reply:
See the below comment - I have some figuring out to do with flow about re-exporting types but I totally agree!